modules/logger.lua

```diff
@@ -138,19 +139,18 @@ end
 ---@field source string source of the log. Usually a playerId or name of a resource.
 ---@field event string the action or 'event' being logged. Usually a verb describing what the name is doing. Example: SpawnVehicle
 ---@field message string the message attached to the log
----@field webhook? string Discord logs only. url of the webhook this log should send to
+---@field webhook? string Discord logs only. url of the webhook this log should send to. looks up via config
```

Comment:
what does looks up via config mean? It's a bit unclear

Reply:
Removed